### DISCUSSION TITLE
Netbox web UI slow for standard users

### DISCUSSION THREAD

**Discussion by CometChasers** [netbox-community/netbox#9346] [Q&A] [answered]
Hi,

We run the latest netbox-docker 1.6.1 and we noticed slowness for standard users while meanwhile, the UI is blazing fast for folks with admin roles. We are talking 5-6 seconds delay for every click in the netbox webUI for normal users and 100-200ms delay for when admin use netbox.

Does anyone knows if there is a known bug in RBAC?

**Comment by RobinBeismann** [comment]
Does anyone have an idea about this? Otherwise I'd raise an issue. This is reproducible across two instances.

**Comment by candlerb** [comment, marked as answer]
No, this is not a known issue in RBAC.  But so far you have not given any information on how to reproduce it; in particular you haven't said anything about what authentication mechanism you're using.

Is it LDAP perhaps? If so, have you tried using tcpdump to see whether multiple queries are being sent each time a non-admin user clicks on something? Is it some mechanism in the front-end proxy with REMOTE_AUTH?  Or is it just plain username and password in the local Netbox auth database?

The other question is, have you set up any permission [constraints](https://docs.netbox.dev/en/stable/administration/permissions/#constraints) which might trigger database queries, directly or indirectly? These checks could be bypassed for users with "superuser" flag set.  Looking at database query logs following clicks may give you clues. Commenting out constraints until you find one which speeds things up may give you clues.

**Reply by RobinBeismann** [reply]
Thanks a lot, those were exactly the ideas I needed, we're using LDAP as backend, I'll check the Authentication logs and a dump next week.

**Reply by RobinBeismann** [reply]
I was able to solve it this way, LDAP Group Caching helped a lot and I was also able to find another configuration issue on the LDAP config.
